src/js/providers/html5.js

```diff
@@ -480,6 +480,10 @@ define([
         }
 
         function _setVideotagSource(item) {
+            if(!item) {
+                return;
```

Comment:
Not good. If this happens we're in big trouble. I'd rather an exception get thrown so we can find the cause and fix it.